### ISSUE TITLE
Is there pursuit environment PP with sparse reward?

### ISSUE THREAD

**Issue by minglou1984** [semitable/seac#5]
Is there any execution code in the pursuit environment PP in the MPE multi-agent environment? I can get the codes of other environments in the paper, except for Predator Pyey.

**Comment by semitable** [comment]
Hi,

I believe it was this environment: https://github.com/semitable/multiagent-particle-envs/blob/master/mpe/scenarios/sparse_predator_prey.py

We also used a pretrained prey and only trained the predators.
